c10/core/TensorImpl.h

```diff
@@ -564,6 +564,22 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
 
   virtual c10::SymIntArrayRef sym_sizes_custom() const;
 
+  // TODO: make it non-virtual after a change to XLA
+  virtual c10::SymInt sym_numel() const {
```

Comment:
I don't see why this has to be virtual. XLA doesn't have this method. So why virtualize it?